control-plane-operator/hostedclusterconfigoperator/controllers/resources/resources.go

```diff
@@ -661,6 +661,10 @@ func (r *reconciler) reconcileOpenshiftOAuthAPIServerEndpoints(ctx context.Conte
 func (r *reconciler) reconcileKubeadminPasswordHashSecret(ctx context.Context, hcp *hyperv1.HostedControlPlane) error {
```

Comment:
can we unit test `reconcileKubeadminPasswordHashSecret`?

Reply:
I couldn’t find a way to get access to that function @enxebre

Reply:
When I first tried I couldn’t get access to the function. Let me double check

Reply:
(Access to call the function from reconcile)

Reply:
Confirmed one does not have access to call that function directly so as is I cannot unit test it

Reply:
I'm not sure how you mean?
In resources_test.go what is preventing us from having
```
func TestReconcileKubeadminPasswordHashSecret() {
  r.reconcileKubeadminPasswordHashSecret
}
```
?

Reply:
My editor was being weird you are right that's fine I'll do that